src/networking.c

```diff
@@ -2716,6 +2720,7 @@ void readQueryFromClient(connection *conn) {
         c->read_reploff += nread;
         atomicIncr(server.stat_net_repl_input_bytes, nread);
     } else {
+        c->net_input_bytes += nread;
```

Comment:
Replicas are clients, is there a reason we wouldn't want to also increment them independently? Otherwise I guess we need to document that the values for replicas are always zero.

Reply:
This metric is for external traffic (i.e. networkr traffic created by users), so I did not count the "internal" traffic. But it is fine to also count replication flow, as we can tell them apart by client flags.

Reply:
let's also count masters and replicas? i suppose it can be useful in some cases.